WordPress/Classes/ViewRelated/NUX/SiteTypeTableViewController.swift

```diff
@@ -87,6 +87,15 @@ class SiteTypeTableViewController: UITableViewController, LoginWithLogoAndHelpVi
         }
     }
 
+    // MARK: - LoginWithLogoAndHelpViewController methods
+
+    func handleHelpButtonTapped(_ sender: AnyObject) {
+        displaySupportViewController(sourceTag: .wpComLogin)
+    }
+    
+    // this VC isn't setup to handle the badge count. Should be fixed when NUXAbstractViewController and LoginViewController are refactored
+    func handleHelpshiftUnreadCountUpdated(_ notification: Foundation.Notification) { }
+
```

Comment:
Hey @nheagy - just wondering why this is. Seems the same as `LoginSocialErrorViewController` - catch the results from `addHelpButtonToNavController` and update an needed. I'm not seeing the difference. Can you elaborate?

Reply:
I had to cut & paste code from `NUXAbstractViewController` into `LoginSocialErrorViewController` to get it to work. I decided not to paste it here because this view isn't yet in production, and I plan to refactor `NUXAbstractViewController` and `LoginWithLogoAndHelpViewController` before this is released.

Sound ok?

Reply:
I see. Yep, that sounds fine. Thanks!